packages/experimental/MenuButton/src/MenuButton.styling.ts

```diff
@@ -0,0 +1,26 @@
+import { UseStylingOptions, buildProps, Theme } from '@fluentui-react-native/framework';
+import { borderStyles, layoutStyles, shadowStyles, fontStyles } from '@fluentui-react-native/tokens';
+import { menuButtonName, MenuButtonProps, MenuButtonSlotProps, MenuButtonTokens } from './MenuButton.types';
+import { defaultButtonTokens, buttonStates } from '@fluentui-react-native/experimental-button/src/ButtonTokens';
+
+export const stylingSettings: UseStylingOptions<MenuButtonProps, MenuButtonSlotProps, MenuButtonTokens> = {
+  tokens: [menuButtonName, defaultButtonTokens],
+  states: buttonStates,
+  slotProps: {
+    button: buildProps(
+      (tokens: MenuButtonTokens, theme: Theme, props: MenuButtonProps) => ({
+        style: {
+          backgroundColor: tokens.backgroundColor,
+          color: tokens.color,
+          iconColor: tokens.iconColor,
+          ...borderStyles.from(tokens, theme),
+          ...layoutStyles.from(tokens, theme),
+          ...shadowStyles.from(tokens, theme),
+          ...fontStyles.from(tokens, theme),
+        },
+        ...props,
```

Comment:
Why are props needed here?

Reply:
We should try to minimize the props we are passing - the more props we pass, the more things we need to check that have changed and the more the cache will be invalidated.

Reply:
In particular the props that should be passed through should only be the ones that are needed by the outer portion of the render function of Button. That means anything that can affect tokens/styling inside slotProps, as well as anything that needs to be passed for useButton to work properly.

Reply:
Thanks for catching it. I destructured needed props. WIll test if it's enough